frontend/src/pages/CourseSelector/CourseSelector.jsx

```diff
@@ -17,6 +17,25 @@ const CourseSelector = () => {
     programCode, programName, specialisation, minor,
   } = useSelector((state) => state.degree);
 
+  const { courses } = useSelector((state) => state.planner);
+
+  useEffect(() => {
+    const openNotification = () => {
+      notification.info({
+        message: "How do I unlock courses on the sidebar?",
+        description: "Quick add courses are made visible when you add their prerequisites to your planner and fulfill UOC requirements. Otherwise all courses can be viewed via the search bar.",
+        duration: 30,
+        className: "text helpNotif",
+        placement: "topRight",
+      });
+    };
+
+    // only open for users with no courses
+    if (Object.keys(courses).length === 0) {
```

Comment:
this depends on courses. add that to the dependency list.

Reply:
ohhh righto sure